### PR TITLE
py310-py311: Update to version 3.10.10 and 3.11.2

### PR DESCRIPTION
**Pull request by th0ma7**
## Description

py310-py311: Update to version 3.10.10 and 3.11.2

Fixes # n/a

## Checklist

- [ ] Build rule `all-supported` completed successfully
- [ ] New installation of package completed successfully
- [ ] Package upgrade completed successfully (Manually install the package again)
- [ ] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Package-Update-Policy#tests-checks)
- [ ] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [ ] Bug fix
- [ ] New Package
- [x] Package update
- [ ] Includes small framework changes
- [ ] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

File: cross/pip/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PKG_NAME = pip
 # use the same version in native/python3/Makefile (python3_native_post_install)
```
Comment (hgy59) [inline cross/pip/Makefile:2]: pip version should be the same in native/python310/Makefile and native/python311/Makefile

Comment (th0ma7) [reply cross/pip/Makefile:2]: good catch, thnx.  done

Comment (hgy59) [reply cross/pip/Makefile:2]: can you adjust the comment in the line above too?

Comment (th0ma7) [reply cross/pip/Makefile]: done

**Comment by th0ma7** [comment]
@hgy59 I noticed homeassistant failing, is this something worth investigating (e.g. unsure what's that package status)?

**Comment by hgy59** [comment]
> @hgy59 I noticed homeassistant failing, is this something worth investigating (e.g. unsure what's that package status)?

The error log shows
```
make[3]: *** [../../mk/spksrc.wheel.mk:139: cross-compile-wheel-orjson] Error 1
```

So it is related to rust and should be fixed.

**Comment by th0ma7** [comment]
> The error log shows
> 
> ```
> make[3]: *** [../../mk/spksrc.wheel.mk:139: cross-compile-wheel-orjson] Error 1
> ```
> 
> So it is related to rust and should be fixed.

@hgy59 Update to latest maturin made orjson to fail building.  Updating it to 3.7.12 (bump +0.0.1) seems to solve things.

**Pull request merged**